sdk/samples/iot/paho_iot_hub_pnp_sample.c

```diff
@@ -33,67 +33,65 @@
 #define SAMPLE_TYPE PAHO_IOT_HUB
 #define SAMPLE_NAME PAHO_IOT_HUB_PNP_SAMPLE
 
-#define TELEMETRY_SEND_INTERVAL 1
-#define TIMEOUT_MQTT_RECEIVE_MAX_MESSAGE_COUNT 3
-#define TIMEOUT_MQTT_RECEIVE_MS (8 * 1000)
-#define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
-#define TIMEOUT_MQTT_WAIT_FOR_COMPLETION_MS 1000
+#define TELEMETRY_SEND_INTERVAL_SEC 1
+#define MQTT_TIMEOUT_RECEIVE_MAX_MESSAGE_COUNT 3
+#define MQTT_TIMEOUT_RECEIVE_MS (8 * 1000)
+#define MQTT_TIMEOUT_DISCONNECT_MS (10 * 1000)
 
-#define DEFAULT_START_TEMP_AVG_COUNT 1
+#define DEFAULT_START_TEMP_COUNT 1
 #define DEFAULT_START_TEMP_CELSIUS 22.0
 #define DOUBLE_DECIMAL_PLACE_DIGITS 2
-#define SAMPLE_PUBLISH_QOS 0
 
 static bool is_device_operational = true;
-static const char iso_spec_time_format[] = "%Y-%m-%dT%H:%M:%S%z"; // ISO8601 Time Format
+static char const iso_spec_time_format[] = "%Y-%m-%dT%H:%M:%S%z"; // ISO8601 Time Format
 
 // * PnP Values *
 // The model id is the JSON document (also called the Digital Twins Model Identifier or DTMI)
 // which defines the capability of your device. The functionality of the device should match what
 // is described in the coresponding DTMI. Should you choose to program your own PnP capable device,
```

Comment:
Unrelated, but:
```suggestion
// is described in the corresponding DTMI. Should you choose to program your own PnP capable device,
```

Reply:
believe it or not, i did just install the vs code spell checker...

Reply:
Be warned, it still misses some things (who knew spell-check on code was such a hard problem) :)